### PR TITLE
docs: remove conjunctions in bullet points

### PR DESCRIPTION
**Pull request by tkoyama010**
Remove conjunctions in bullet points.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ pyOpenSci is devoted to building diverse, supportive community around
 the Python open source tools that drive open science. We do this through:
 
 * open peer review
-* mentorship and
-* training.
+* mentorshipd
```
Comment (tkoyama010) [inline README.md:18]:
```suggestion
* mentorship
```

**Review by lwasser — APPROVED** [review]
@tkoyama010 this is opened as a draft PR but it looks to me like we can merge. Should we go ahead and merge it? i'm approving it now.

**Comment by tkoyama010** [comment]
Thanks! Let's merge.

**Comment by lwasser** [comment]
perfect!!

**Pull request merged**